wasp-ai/main.wasp

```diff
@@ -154,7 +159,7 @@ entity SocialLogin {=psl
   providerId String
 
   userId Int
-  user User @relation(fields: [userId], references: [id])
+  user User @relation(fields: [userId], references: [id], onDelete: Cascade)
```

Comment:
Thought: once projects are deleted, we will never know they existed, so our number of created projects will become smaller. Would be cool if they only thing that is left is this idea of "there was a project that was deleted". But to be honest I am not sure myself how to implement that simply, so probably best to just ignore it, doesn't matter.

Reply:
this isnt for projects. when the user is deleted it deletes the social login entity, but leaves their projects in the DB.

Reply:
I think it's fine to leave the generated apps in the database in case they've been shared. there is no connection whatsoever to the user that created it.

Reply:
ok I've gone ahead and deleted all user-relevant info leaving just the project shell :)